tests/char_volume_test.cpp

```diff
@@ -44,18 +44,23 @@ TEST_CASE( "character_baseline_volumes", "[volume]" )
     Character &you = get_player_character();
     REQUIRE( you.get_mutations().empty() );
     REQUIRE( you.height() == 175 );
+    CHECK( ( you.get_base_volume() > 55_liter && you.get_base_volume() < 80_liter ) );
 
     REQUIRE( your_height_with_trait( trait_SMALL2 ) == 70 );
-    CHECK( your_volume_with_trait( trait_SMALL2 ) == 23326_ml );
+    CHECK( ( your_volume_with_trait( trait_SMALL2 ) > 20_liter &&
+             your_volume_with_trait( trait_SMALL2 ) < 30_liter ) );
```

Comment:
I don't see any reason to check within a range, any changes to volume should always be caught by this test. Anyone updating the volume calculations should also be capable of updating the test values - if not, *they really shouldn't be changing the volume calculations*.

Reply:
I'll fix that :-)